src/tooling/docs-assembler/assembler.yml

```diff
@@ -98,8 +88,37 @@ references:
   elastic-otel-node:
   elastic-otel-php:
   elastic-otel-python:
-  eland:
   elastic-serverless-forwarder:
+  logstash-docs-md:
+  logstash:
+    current: "9.0"
+    next: main
+  opentelemetry:
+
+  # @elastic/ingest-docs and @elastic/experience-docs
```

Comment:
```suggestion
  # @elastic/experience-docs
```

Reply:
I'll make add a commit that moves these up into ingest-docs (in alphabetical order).